conf/local-config.json

```diff
@@ -20,9 +20,13 @@
   "admin_key_prefix": "ADINT",
   "master_key_activates_in_seconds": 30,
   "master_key_expires_after_seconds": 60000,
+  "master_key_rotation_cut_off_days": 37,
```

Comment:
Why 37?

Reply:
https://atlassian.thetradedesk.com/confluence/display/EN/UID2-599+Delete+keys+that+has+expired+for+more+than+30+days+during+key+rotation

Here is the detailed explanation! 

Short version is:

- refresh key cut off = expiry timestamp + refresh token lifetime (30d) + troubleshooting buffer (7d)
- master key cut off = expiry timestamp + max advertising token lifetime (30d) + troubleshooting buffer (7d)
- site key cut off = expiry timestamp + max advertising token lifetime (30d) + troubleshooting buffer (7d)

Reply:
Uh. The page has some mistakes (like refresh_key lifetime is definitely not 1h) but your analysis remains correct. So 30d is the estimated maximum lifetime given by UID Sharing task force. Thanks!